Dijkstra_Algorithm/Dijkstra_Algorithm.dart

```diff
@@ -0,0 +1,96 @@
+/*
+ Djikstra's algorithm (named after its discover, E.W. Dijkstra) solves the
+ problem of finding the shortest path from a point in a graph (the source)
+ to a destination.
+ It turns out that one can find the shortest paths from a given source to all
+ points in a graph in the same time, hence this problem is sometimes called
+ the single-source shortest paths problem.
+*/
+int mind(dist, visited , n)
```

Comment:
Please give meaning function name. The name mind is not clear about the purpose of this function.